qmaftools/src/org/qcmg/maf/MafFilter.java

```diff
@@ -138,7 +127,7 @@ private void loadKRASFile(String krasFile) throws Exception {
 								logger.info("verification DOES NOT match! - updating");
 								
 								// update record with "Valid" validation status
-								tr.setData(tr.getData().replaceAll("Unknown", "Valid"));
+								tr = tr.replaceAll("Unknown", "Valid");
```

Comment:
this is not right - you are reassigning the pointer to a new object, the collection storing the strings is not updated

Reply:
yes, it was a mistake and fixed. Thanks